src/browser/index.js

```diff
@@ -42,17 +43,40 @@ if (isReload) {
 document.body.classList.toggle('use-native-titlebar',
                                Runtime.useNativeTitlebar());
 
+const replay =
+  () => {
+    const request = new XMLHttpRequest();
+    // We use sync XHR as we have nothing else to render otherwise.
```

Comment:
> What's the downside of using an async xhr?

I would just have to wrap most of the what happens in this file in a function and call it on response. Also as I recall at least in gecko we had to draw something right away.

It's also kind of just debug tool and there is not much we can do until we get response so...

I can change it to use async if you want.